.github/workflows/check-comment-reply.yml

```
@@ -0,0 +1,75 @@
+          import requests
+          from collections import defaultdict
+
+          headers = {
+              "Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}",
+              "Accept": "application/vnd.github+json"
+          }
+
+          repo = os.environ['GITHUB_REPOSITORY']
+          pr_number = os.environ.get('GITHUB_REF').split('/')[-1]
+          api_url = f"https://api.github.com/repos/{repo}/issues/{pr_number}/comments"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)